app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,5 @@
+class WelcomeController < ApplicationController
+  def index
+    
```

Comment:
We need to discuss how to package the stats we need and send them through to the view.